pl/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Elixir School
+lang: pl
+---
+
+[![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+
+Lekcje programowania w języku Elixir, inspirowane lekcjami Twittera [Scala School](http://twitter.github.io/scala_school/). 
+
+Dostepne w [Việt ngữ][vi], [汉语][cn], [Español][es], [日本語][jp] oraz [Português][pt].
```

Comment:
We're thinking about removing this line from the indexes with the exception of English, what do you think?  See #270